.gitignore

```diff
@@ -25,13 +26,9 @@ gen-docs/
 thrift_api/gen-js/
 thrift_api/gen-py/
 
-# test_files build files
-tests/test_projects/test_files/call_and_message.o
-tests/test_projects/test_files/divide_zero.o
-tests/test_projects/test_files/new_delete.o
-tests/test_projects/test_files/null_dereference.o
-tests/test_projects/test_files/stack_address_escape.o
-tests/test_projects/test_files/file_to_be_skipped.o
+# object and binary files for tests
```

Comment:
With the splitting up of the Makefile i agreed because only the testing part and configuration is ~50 lines which is almost the same size as the main `Makefile`.
In the `.gitignore` there are only 2 lines which are specific for test files and like to see at one place what is ignored from the repository.